consensus/src/consensus/utxo_set_override.rs

```diff
@@ -0,0 +1,34 @@
+#[cfg(feature = "developer-mode")]
+pub mod utxo_set_override {
+    use std::sync::Arc;
+
+    use itertools::Itertools;
+    use kaspa_consensus_core::{
+        api::ConsensusApi, config::Config, header::Header, muhash::MuHashExtensions, utxo::utxo_collection::UtxoCollection,
+    };
+    use kaspa_hashes::Hash;
+    use kaspa_muhash::MuHash;
+
+    use crate::consensus::Consensus;
+
+    pub fn set_genesis_utxo_commitment_from_config(config: &mut Config) {
+        let mut genesis_ms = MuHash::new();
```

Comment:
I prefer `genesis_multiset`. `ms` is automatically interoperated as milliseconds